tests/system/action/user/test_forget_password.py

```diff
@@ -0,0 +1,73 @@
+from time import time
+
+from openslides_backend.action.mixins.send_email_mixin import (
+    ConnectionSecurity,
+    EmailSettings,
+)
+from tests.system.action.base import BaseActionTestCase
+from tests.system.action.mail_base import AIOHandler, AiosmtpdServerManager
+
+
+class UserForgetPassword(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        EmailSettings.connection_security = ConnectionSecurity.NONE
+        EmailSettings.host = "127.0.0.1"
+        EmailSettings.port = 25
+        EmailSettings.timeout = 5
+        EmailSettings.user = ""
+        EmailSettings.password = ""
+        EmailSettings.accept_self_signed_certificate = False
+        EmailSettings.default_from_email = "noreply@example.com"
```

Comment:
Also, please extract this into a helper function since it is used in other tests as well.

Reply:
Add a helper function. It is also used in test_send_invitation_mail, use there the helper too.